src/cursor/cur_table.c

```diff
@@ -433,22 +439,23 @@ __curtable_prev(WT_CURSOR *cursor)
 static int
 __curtable_reset(WT_CURSOR *cursor)
 {
-    WT_CURSOR *primary;
+    WT_CURSOR **cp;
     WT_CURSOR_TABLE *ctable;
     WT_DECL_RET;
     WT_SESSION_IMPL *session;
+    u_int i;
 
     ctable = (WT_CURSOR_TABLE *)cursor;
-    /* Grab the primary cursor to reset the bounds. */
-    primary = *ctable->cg_cursors;
 
     JOINABLE_CURSOR_API_CALL_PREPARE_ALLOWED(cursor, session, reset, NULL);
 
     APPLY_CG(ctable, reset);
 
-    /* If a user calls cursor reset also reset the bounds. */
-    if (API_USER_ENTRY(session))
```

Comment:
np: no need for braces

Reply:
[5b20368](https://github.com/wiredtiger/wiredtiger/pull/8265/commits/5b203680c47ab284e59883d194495a9a6703223e)